canonical-web-and-design/snapcraft.io
# Tour content

Fixes #2199 
Fixes #2200 

Adds full content of publisher snap listing page tour.
Moves tour content from template to YAML file.
Makes it possible to target multiple elements with one tour step for better control over what is highlighted on the page.

### QA
- ./run or https://snapcraft-io-canonical-web-and-design-pr-2213.run.demo.haus/
- sign in and go to publisher listing page of any snap
- click on a tour icon (?) in right bottom corner
- go through the tour
- make sure it works fine when scrolling, resizing, etc
- try different snaps (with different amount of data filled in already)

<img width="1098" alt="Screenshot 2019-08-27 at 09 19 23" src="https://user-images.githubusercontent.com/83575/63750078-0dec5400-c8ad-11e9-9a0f-7db33e976f1c.png">
